devel/ccache/Portfile

```diff
@@ -27,6 +29,7 @@ homepage            https://ccache.dev
 
 # gcc 4.x clang < 500 does not understand -Wpedantic
 pre-configure {
+    delete ${worksrcpath}/dev_mode_disabled
```

Comment:
Why remove this file ? Can you add a comment explaining why ?
Please also move the Wpedantic comment to just before the reinplace line

Reply:
I neglected to ask what should be done here.

The GitHub portgroup recommends using manually-created `releases` distfiles rather than the automatically created `archive`/`tarball`. However, this port wants to run ./autogen.sh, which ccache believes is unnecessary for release distfile users; anyone who wants to run ./autogen.sh anyways first must remove the dev_mode_disabled file.

Should this instead continue using `tarball_from tarball` or `archive`? Then there would be no dev_mode_disabled to remove. But I would probably then also add a comment explaining why the `releases` distfile is *not* preferable in this case (because the port runs ./autogen.sh which is "unsupported"/discouraged by the `releases` distfile.).

Reply:
I have switched it back to using `tarball`, though it now does so explicitly, and there is a comment explaining why `releases` isn't used.